src/qml/SearchBar.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick
+import QtQuick.Controls
+import org.qfield
+import Theme
+
+Rectangle {
+  z: 20
+
+  signal validateFilter(string text)
```

Comment:
Hmm, ok, the name here isn't convincing me ;) _if_ we keep a signal, maybe searchTermChanged is a better name.

But, I think here we should use a property alias searchTerm: searchBar.text ; it's much better to expose this as a property than a signal.